### PR TITLE
TA-84144

### PR DESCRIPTION
**Pull request by gemma-shay**
https://cockroachlabs.atlassian.net/browse/DOC-5008

### CODE REVIEW

**Review by postamar — APPROVED** [review]
a84144.md LGTM

**Comment by gemma-shay** [comment]
> a84144.md LGTM

Looks like changes from another PR got pulled into this somehow.. will try to revert

File: advisories/a84144.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+affected_versions: v22.1.0-v22.1.3
+advisory_date: 2022-07-18
+docs_area: releases
+---
+
+Publication date: {{ page.advisory_date | date: "%B %e, %Y" }}
+
+## Description
+
+The problem affects multi-region tables whose locality has been altered to `REGIONAL BY ROW` in clusters running versions v22.1.0 to v22.1.3. The problem occurs when dropping a region: the drop may proceed when instead it should have yielded an error. Any subsequent attempt at changing the locality of the table will fail with an internal error.
```
Comment (rmloveland) [inline advisories/a84144.md:15]: Suggest adding links to:

-  info re: `REGIONAL BY ROW` tables (eg https://www.cockroachlabs.com/docs/stable/multiregion-overview#regional-by-row-tables)
- v22.1 release notes page
- DROP REGION docs: https://www.cockroachlabs.com/docs/v21.1/drop-region.html

File: advisories/a84144.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+~~~ sql
+> ALTER TABLE kv SET LOCALITY GLOBAL;
+ERROR: failed to construct index entries during backfill: error decoding 4 bytes: could not find [128] in enum "public.crdb_internal_region" representation PhysicalReps: [[64]]; LogicalReps: [us-east1] goroutine 45657 [running]:
+~~~
+
+## Statement
+
+This is resolved in CockroachDB by PRs [#84339](https://github.com/cockroachdb/cockroach/pull/84339) and [#84349](https://github.com/cockroachdb/cockroach/pull/84349) which ensure, respectively, that:
+back-references to tables using the region enum type are correctly updated in the region enum type descriptor when performing `ALTER TABLE … SET LOCALITY REGIONAL BY ROW;` any existing type descriptors which are already missing these back-references are updated prior to `ALTER DATABASE … DROP [SUPER] REGION` so as to ensure its correct behavior.
```
Comment (rmloveland) [inline advisories/a84144.md:44]: region enum could link to https://www.cockroachlabs.com/docs/v21.1/set-locality#crdb_region, I think

**Review by rmloveland — APPROVED** [review]
LGTM, just annoying you about links

File: advisories/a84144.md (hunk 2 above)
Comment (gemma-shay) [reply advisories/a84144.md]: thanks!

**Pull request merged**